openmc_plotter/plotmodel.py

```diff
@@ -650,9 +700,41 @@ def _do_op(array, tally_value, ax=0):
 
         return image_data, extents, data_min, data_max
 
+class ViewParam(openmc.lib.plot._PlotBase):
+    def __init__(self, origin=(0, 0, 0), width=10, height=10):
+        super().__init__()
+        # View Parameters
+        self.level = -1
+        self.origin = origin
+        self.width = width
+        self.height = height
+        self.h_res = 1000
+        self.v_res = 1000
+        self.basis = 'xy'
+        self.color_overlaps = False
```

Comment:
this `color_overlaps` property was not actually present in the original class, but it needed to be explicitly for `__getattr__`/`__setattr__` to work.